app/routables/index/styles.scss

```diff
@@ -1,168 +1,307 @@
-            a {
-              color: white;
-              text-decoration: underline;
-              &:hover {
-                text-decoration: none;
-                color: #ffdb5b;
-              }
-            }
+        div::before,
+        div::after {
+          top: 0;
+          transform: rotate(90deg);
+        }
+        .toggler:checked ~ .menu {
+          visibility: visible;
+          div {
+            transform: scale(1);
+            transition-duration: 0.8s;
           }
         }
       }
-    }
-  }
-}
+      .menu {
+        position: fixed;
+        top: 0px;
+        right: 0px;
+        width: 100%;
+        height: 100%;
+        visibility: hidden;
+        overflow: hidden;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .one {
+        background: #040009;
+        border-radius: 50%;
+        width: 200vw;
+        height: 200vw;
+        display: flex;
+        flex: none;
+        align-items: center;
+        justify-content: center;
+        transform: scale(0);
+        transition: all 0.4s ease;
+      }
+      .two {
+        text-align: center;
+        max-width: 90vw;
+        max-height: 100vh;
+        opacity: 0;
+        transition: opacity 0.4s ease;
+      }
+      ul {
+        li {
+          list-style: none;
+          color: black;
+          font-size: 45px;
+          padding: 40px;
 
-section#contact {
-  background: #252525;
-  padding: 60px;
-  div {
-    text-align: center;
-    margin: 25px;
-    p {
-      color: white;
-      font-size: 20px;
-      line-height: 150%;
-      a {
-        color: white;
+          a {
+            text-decoration: none;
+            transition: color 0.4s ease;
+            color: #fff;
+          }
+        }
       }
     }
-  }
-}
-
-section#founders {
-  ul {
-    display: flex;
-    background-color: white;
-    padding: 25px;
-    li {
-      flex: 1;
-      margin: 25px;
-      text-align: center;
-      img {
+    .hamburger {
+      position: absolute;
+      top: 20px;
+      right: 80px;
+      z-index: 1;
+      width: 60px;
+      height: 60px;
+      padding: 10px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      > div {
+        position: relative;
         width: 100%;
-        height: auto;
-        padding: 20px;
-      }
-      h3 {
-        color: #252525;
-        padding: 10px;
+        height: 2px;
+        background-color: #fff;
+        display: flex;
+        align-item: center;
+        justify-content: center;
+        transition: all 0.4s ease;
       }
-      h4 {
-        color: #252525;
+      > div::before,
+      ::after {
+        content: "";
+        position: absolute;
+        z-index: 1;
+        top: -10px;
+        width: 100%;
+        height: 2px;
+        background: white;
       }
-      p {
-        color: #252525;
+      > div::after {
+        top: 10px;
       }
     }
   }
 }
+
+// main#homepage.page {
```

Comment:
Consider removing big blocks of unused code.

Reply:
Yes, I think once the site is all done, ill remove what I didnt have, kind of want to keep some incase I want to re-introduce.